src/index.js

```diff
@@ -1,13 +1,193 @@
 export default function BaseballGame() {
-  this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
-  };
-}
+  let _computerInputNumber = getComputerInputNumber();
+  let _retryCount = 0;
+
+  addButtonEvent(
+    `submit${_retryCount === 0 ? '' : _retryCount}`,
+    clickSubmitButton,
+  );
+
+  return;
+
+  /*
+   ** Play Game
+   */
+
+  function getResult({ strikeCount, ballCount }) {
+    let $result = document.getElementById('retryResult');
+    if (!$result) {
+      $result = document.getElementById('result');
+      $result.innerHTML += `<div id="retryResult"></div>`;
+      $result = document.getElementById('retryResult');
```

Comment:
`Element.insertAdjacentHTML()` 같은 메서드를 사용하면 더 깔끔하게 표현할 수 있을거 같아요!

```suggestion
      $result = document.getElementById('result');
      $result.insertAdjacentHTML('beforeend', `<div id="retryResult"></div>`)
```

- [Element.insertAdjacentHTML() / MDN](https://developer.mozilla.org/ko/docs/Web/API/Element/insertAdjacentHTML)

Reply:
오오..! 이런게 있었군요! 감사합니다!